apollo-router/src/services/mod.rs

```diff
@@ -608,29 +639,18 @@ impl ExecutionRequest {
     }
 }
 
-assert_impl_all!(ExecutionResponse: Send);
+//assert_impl_all!(ExecutionResponse: Send);
```

Comment:
Probably remove rather than just commenting out.

Reply:
added back in [440cdc0](https://github.com/apollographql/router/pull/1206/commits/440cdc00f5109c0843e70ad0ade050e4b07bde36)